app.py

```diff
@@ -1,41 +1,32 @@
-"""Copyright (c) 2022 VIKTOR B.V.
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
-documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
-rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
-persons to whom the Software is furnished to do so, subject to the following conditions:
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
-Software.
-VIKTOR B.V. PROVIDES THIS SOFTWARE ON AN "AS IS" BASIS, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
-NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT
-SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF
-CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
-"""
-
-from io import StringIO, BytesIO
+from io import BytesIO
+from io import StringIO
 from pathlib import Path
 
-import lxml
-
-from owslib.wms import WebMapService
+from viktor import UserError
+from viktor import ViktorController
+from viktor.parametrization import MultiSelectField
+from viktor.parametrization import OptionField
+from viktor.parametrization import SetParamsButton
+from viktor.parametrization import Step
+from viktor.parametrization import Text
+from viktor.parametrization import TextAreaField
+from viktor.parametrization import ViktorParametrization
+from viktor.result import SetParamsResult
+from viktor.views import DataGroup
+from viktor.views import DataItem
+from viktor.views import DataResult
+from viktor.views import DataView
+from viktor.views import WebResult
+from viktor.views import WebView
 
+# folium and owslib
 import folium
+import lxml
 from folium import plugins
```

Comment:
I don't think this needs a separate import for a single use. You can replace `plugins.Draw` with `folium.plugins.Draw` in the app code

Reply:
done!